src/Microsoft.AspNet.Mvc.Core/ConsumesAttribute.cs

```diff
@@ -53,13 +64,12 @@ public void OnResourceExecuting(ResourceExecutingContext context)
             // Ignore all other filters. This is to ensure we have a overriding behavior.
             if (IsApplicable(context.ActionDescriptor))
             {
-                MediaTypeHeaderValue requestContentType = null;
-                MediaTypeHeaderValue.TryParse(context.HttpContext.Request.ContentType, out requestContentType);
+                var requestContentType = context.HttpContext.Request.ContentType;
 
                 // Confirm the request's content type is more specific than a media type this action supports e.g. OK
                 // if client sent "text/plain" data and this action supports "text/*".
                 if (requestContentType != null &&
```

Comment:
Agree there's no downside to leaving this code as-is because you've made `IsSubsetOf()` resilient.

----

FYI
>  `requestContentType` is not a property that a user will normally set

Huh? That variable comes directly from the HTTP request's headers. And that header is set in almost-all POST requests.